db/migrate/20200801141038-modify_payment_add_paid_at.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
```

Comment:
Please delete this file, as it is not being used. It seems you generated the migration twice